docs/cli.md

```diff
@@ -169,75 +177,75 @@ gcli metalake list
 #### Show a metalake details
 
 ```bash
-gcli metalake details --metalake metalake_demo
+gcli metalake details
 ```
 
 #### Show a metalake audit information
 
 ```bash
-gcli metalake details --metalake metalake_demo --audit
+gcli metalake details --audit
 ```
 
 #### Create a metalake
 
 ```bash
-gcli metalake create --metalake my_metalake --comment "This is my metalake"
+gcli metalake create --comment "This is my metalake"
```

Comment:
for the create command, I guess the "--metalake" option is required?

Reply:
Yes it is - again that is in another PR that needs to be reviewed first.

Reply:
I know it doesn't have to be, but as a good example (and in a real case), user would specify it inline here, instead of directly configured in the configure file or environment variable. Otherwise, this sample looks strange.

Reply:
I've added it back in, but it is not required